packages/block-library/src/index.native.js

```diff
@@ -288,7 +288,7 @@ export const registerCoreBlocks = () => {
  * @constant {{ string, number }}
  */
 export const NEW_BLOCK_TYPES = {
-	[ embed.name ]: 3,
-	[ search.name ]: 3,
-	[ audio.name ]: 3,
+	[ embed.name ]: 10,
```

Comment:
10 feels a bit low to me. I'd lean toward something in the 20-40 range myself. No strong feelings though, just a thought and certainly not a blocker.

Reply:
After reconsidering, I definitely agree. The block inserter likely is a high-touch UI element. I have increased the default impression count to 40. Addressed in https://github.com/WordPress/gutenberg/pull/33906/commits/5bf9abfe84777565e64f43822e9378f376456580.